dotNet/Institution/Program.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+
+namespace Institution
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("-- Abstract Implementation --");
+            StudentA studentA = new StudentA("female", "6", "student", "Аня", "1A" );
+            TeacherA teacherA = new TeacherA("female", "25", "teacher", "Светлана Ивановна", "1A" );
```

Comment:
Если нету реальной причины использовать явную типизацию, то стоит использовать неявно "var"
```suggestion
            var studentA = new StudentA("female", "6", "student", "Аня", "1A" );
            var teacherA = new TeacherA("female", "25", "teacher", "Светлана Ивановна", "1A" );
```